index-style.css

```diff
@@ -0,0 +1,63 @@
+    body{
+        background-color:#deedec;
+        color:#000000;
+    }
+
+    ul{
+        font-family:'segoe ui light';
+        list-style-type: none;
+        padding:0;
+    }
+
+    li{
+        float:left;
+        }
+
+    a{
+        display:inline-block;
+        padding-right:75px;
+        padding-left:75px;
+        padding-top:10px;
+        padding-bottom: 10px;
+        background-color:#ffffff;
+        border-radius:25px;
+        margin-left: 5px;
+        margin-right:5px;
+    }
+
+    h1{
+        color:#000000;
+        font-family:Castellar;
```

Comment:
I already mentioned Google fonts, but another thing you generally want to do is give a fallback font. You could say
```
font-family:Castellar, serif
```
That way if they don't have Castellar, it would default to the computer's standard serif font (I'm not sure if Castellar is serif or not, that's just an example).